nvdaHelper/cppjieba/cppjieba.hpp

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
same here in regards to https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/copyrightHeaders.md